README.md

```diff
@@ -1,12 +1,14 @@
 # Business Site
 
-Replace this readme with your own information about your project. 
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Build a responsive web site for a business, containg a hero with backround picture, an accordion with the help of Javascript and a form. 
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+As last project I went ahead of myself and had to make myself stop and start over. One thing at the time. And after I got stuck with the Accordion I let it go for a While and set up the hero and form. 
```

Comment:
I totally understand you! All these beautiful works were achieved in just 3 weeks. We have bunch of time ahead of us, we will learn to manage time better by doing. We've come a long way so far. Let's keep learning!

Reply:
You say it so well! Thank you for your review! Really useful comments to take with me! ⭐️♥️